tests/test_waku_lightpush.nim

```diff
@@ -115,4 +116,40 @@ suite "Waku Lightpush":
       requestError == error
 
     ## Cleanup
-    await allFutures(clientSwitch.stop(), serverSwitch.stop())
\ No newline at end of file
+    await allFutures(clientSwitch.stop(), serverSwitch.stop())
+
+  asyncTest "request with an empty content topic should fail":
+    ## Setup
+    let
+        serverSwitch = newTestSwitch()
+        clientSwitch = newTestSwitch()
+
+    await allFutures(serverSwitch.start(), clientSwitch.start())
+
+    ## Given
+    let error = emptyContentTopicFailure
+    
+    let handler = proc(peer: PeerId, pubsubTopic: PubsubTopic, message: WakuMessage): Future[WakuLightPushResult[void]] {.async.} =
+        ## this handler will never be called: request will fail due to empty content topic
+        return ok()
+
+    let
+      server = newTestWakuLightpushNode(serverSwitch, handler)
+      client = newTestWakuLightpushClient(clientSwitch)
+      serverPeerId = serverSwitch.peerInfo.toRemotePeerInfo()
+      message = fakeWakuMessage(contentTopic="")
```

Comment:
The bug is that an empty response is received, not on content topic being an empty string, but that the Waku Message and PushRPC is constructed without this field and decoding fails.